content/posts/2022/retention-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Automatically delete workspace files with retention rules"
+date: 2022-11-29T10:30:00+01:00
+---
```

Comment:
Our current [tag cloud](https://blog.reana.io/tags/) looks kind of sad, I guess we can consider removing tags from all the posts?  (Otherwise you could use `#workspace #quota` perhaps, but removing all the tags looks cleaner.)

Reply:
I have removed tags from all the posts